Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -439,15 +445,13 @@ - (nullable UIColor *)calendar:(FSCalendar *)calendar
                     appearance:(FSCalendarAppearance *)appearance
        fillDefaultColorForDate:(NSDate *)date {
     if ([self isDateEnabled:date]) {
-        if ([self.delegate respondsToSelector:@selector(calendar:cellStyleForDate:)]) {
+        if ([self.delegate respondsToSelector:@selector(calendar:cellDataForDate:)]) {
             BPKSimpleDate *simpleDate = [[BPKSimpleDate alloc] initWithDate:date forCalendar:self.gregorian];
 
-            BPKCalendarDateCellStyle style = [self.delegate calendar:self cellStyleForDate:simpleDate];
+            id<BPKCalendarCellData> cellData = [self.delegate calendar:self cellDataForDate:simpleDate];
 
-            // For the custom style we fallback to calling
-            // `calendar:fillColorForDate:` below
-            if (style != BPKCalendarDateCellStyleCustom) {
```

Comment:
Doesn't RN rely on this still?

Reply:
Yeah but I guess we can migrate it to instead return one of the instances of `BPKCalendarTrafficLightCellData`?

Reply:
We should verify that this will indeed be sufficient

Reply:
I can't see any reason why not.
It also uses the `DONT_USE_calendar` delegate methods but those are still intact